src/BroadcastMsg.chpl

```diff
@@ -134,6 +140,46 @@ module BroadcastMsg {
     return new MsgTuple(repMsg, MsgType.NORMAL);    
   }
 
+  proc broadcastStrings(cmd: string, msgArgs: borrowed MessageArgs, st: borrowed SymTab): MsgTuple throws {
+    const gs = getGenericTypedArrayEntry(msgArgs.getValueOf("segName"), st);
+    const segs = toSymEntry(gs, int);
+    // Check that values exists (can be any dtype)
+    const sE = getSegString(msgArgs.getValueOf("valName"), st);
```

Comment:
Not sure this comment fully makes sense here. This looks more like we are just accessing values, not checking existence. 

Also, calling `getSegString` with this comment is not straight forward because this would imply that values must be a `SegString` not any dtype.

@pierce314159  - Can you provide some clarification here?

Reply:
I won't lie, that comment is from the original PR and it looks like it was copied from here
https://github.com/Bears-R-Us/arkouda/blob/732c48fea6f2392b25f554dfda59b7eca10b8d6c/src/BroadcastMsg.chpl#L33-L34

I agree it doesn't need to be there! I'll go ahead and remove